tests/index.html

```diff
@@ -13,8 +13,16 @@
     <link rel="stylesheet" href="assets/vendor.css">
     <link rel="stylesheet" href="assets/dummy.css">
     <link rel="stylesheet" href="assets/test-support.css">
-
     {{content-for "head-footer"}}
+    <script type="text/javascript"
```

Comment:
As above, I think installing MathJax via bower should be a separate ticket.  I experimented for a little while and it looked like it was going to take some work.  Preprints and Registries are currently using CDN - this copied bit was just so I could get the dummy app to look like preprints.